app/access_grant_funding/templates/access_grant_funding/base.html

```diff
@@ -53,7 +53,11 @@
           {
             "href": config.ACCESS_SERVICE_DESK_URL,
             "text": "Contact us"
-          }
+          },
+          {
+            "href": url_for("access_grant_funding.cookies"),
+            "text": "Cookies"
+          },
```

Comment:
nit: Probably makes sense to group privacy+cookies+accessibility together, rather than split around the contact link